catalog-info.yaml

```diff
@@ -100,11 +143,11 @@ spec:
     apiVersion: buildkite.elastic.dev/v1
     kind: Pipeline
     metadata:
-      name: docs-test
+      name: docs / test
     spec:
-      branch_configuration: "main"
+      branch_configuration: master
       repository: elastic/docs
-      pipeline_file: ".buildkite/pr_main_mr_test.yml"
+      pipeline_file: ".buildkite/test_pipeline.yml"
```

Comment:
It doesn't look like the `.buildkite/scripts/test.sh` script that this pipeline calls is executable. Maybe Buildkite handles this automatically, but it might help to change this to explicitly use `bash`, like L4 of `.buildkite/build_pr_pipeline.yml`